modules/ROOT/pages/dataweave-cookbook-map-an-object.adoc

```diff
@@ -4,11 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, json, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, as
 
-
-This DataWeave example uses both the `map` and `mapObject` functions to iterate through the input and set all of the keys to upper case.
+The following DataWeave examples use the `mapObject` function to iterate through the keys and values in each of the objects of the arrays.
```

Comment:
```suggestion
The following DataWeave examples use the `mapObject` function to iterate through the keys and values of objects.
```
It does not iterate over arrays. The array are transformed by other functions to an object and the mapObject iterate through that object.